etl/steps/data/garden/owid/2022/living_planet_index.meta.yml

```diff
@@ -0,0 +1,26 @@
+dataset:
+  title: Living Planet Index – WWF (2022)
+tables:
+  living_planet_index:
+    variables:
+      living_planet_index_average:
+        title: living_planet_index_average
+        description: The Living Planet Index (LPI) is a measure of the state of global biological diversity based on population
+          trends of vertebrate species from around the world. The index value is measured relative to species' populations
+          in 1970 (i.e. 1970 = 1).
+        unit: (1970 = 1)
+        display:
+          includeInTable: true
+          name: Living Planet Index
+      living_planet_index_lower:
+        title: living_planet_index_lower
+        unit: (1970 = 1)
```

Comment:
Will this overwrite the chart? There is an conversion factor of 100 on the chart to turn it into a %, I'm not sure whether the etl or chart edits take precedence in cases like this?

Reply:
Chart config always takes precedence, so we should be good here.